src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapper.cs

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer.Interfaces
 {
     using System.Collections.Generic;
+    using System.Threading.Tasks;
```

Comment:
I think it's appropriate to have both async and non-async methods available to consumers, particularly since console applications are one of the more common uses of the platform. However, splitting the methods into async and non-async interfaces, and having the concrete class implement both, sounds like an excellent idea. I kind of hate the idea of anything named MyThing2, but this isn't my codebase. :)